docs/maui/TOC.yml

```
@@ -181,6 +181,8 @@ items:
       href: imagesources/GravatarImageSource.md
 - name: "Layouts"
   items:
+    - name: "DockLayout"
+      href: layouts/DockLayout.md
     - name: 'StateContainer'
```

This isn't related to DockLayout, but I noticed that `StateContainer` uses single quotes instead of double quotes

```suggestion
    - name: "StateContainer"
```